application-diagram-ui/src/main/resources/Diagram/DiagramEditSheet.xml

```diff
@@ -554,6 +554,7 @@ define('diagram-link-editor', [
   'draw.io',
   'resourceSelector'
 ], function($, diagramLinkHandler) {
+    window.mxIsElectron = false;
```

Comment:
Could this be moved in https://github.com/xwikisas/application-diagram/blob/master/application-diagram-ui/src/main/resources/Diagram/DiagramSheet.xml#L247  , where other set-up values are added?